src/it/scala/com/wavesplatform/it/RollbackSpec.scala

```diff
@@ -0,0 +1,69 @@
+package com.wavesplatform.it
+
+import org.scalatest.{FreeSpec, Matchers}
+import org.scalatest.concurrent.{IntegrationPatience, ScalaFutures}
+
+import scala.collection.mutable
+import scala.concurrent.Await.result
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+import scala.concurrent.Future.traverse
+import scala.concurrent.duration._
+import scala.collection.JavaConverters._
+
+class RollbackSpec extends FreeSpec with ScalaFutures with IntegrationPatience
```

Comment:
Basically the difference between two tests are
```
 +      _ <- traverse(nodes)(_.rollback(startHeight, returnToUTX = false))
 +      _ <- processRequests(requests)
```
vs 
```
 +      _ <- traverse(nodes)(_.rollback(startHeight))
```
Might be reasonalbe to refactor that.